packages/client/hmi-client/src/composables/useCharts.ts

```diff
@@ -40,30 +55,61 @@ interface ChartData {
  */
 export function useCharts(
 	nodeId: string,
-	model: Ref<Model | null>,
-	modelConfig: Ref<ModelConfiguration | null>,
+	model: Ref<Model | null> | null,
+	modelConfig: Ref<ModelConfiguration | null> | null,
 	chartData: Ref<ChartData | null>,
 	chartSize: Ref<{ width: number; height: number }>,
-	interventions?: Ref<Intervention[]> | null
+	interventions: Ref<Intervention[]> | null,
+	mapping: Ref<VariableMappings> | null
 ) {
 	// Setup annotations
 	const { getChartAnnotationsByChartId, generateAndSaveForecastChartAnnotation } = useChartAnnotations(nodeId);
 
 	const getUnit = (paramId: string) => {
-		if (!model.value) return '';
+		if (!model?.value) return '';
 		return getUnitsFromModelParts(model.value)[paramId] || '';
 	};
 
+	const createEnsembleVariableChartOptions = (setting: ChartSettingEnsembleVariable) => {
```

Comment:
I see this has pre and post run labels
Is this specific to calibration? It may help to rename

Reply:
We use `:pre` label for the base data for all non ensemble calibrate, optimize (before and after optimize), and simulation (with or without intervention). I found it's a bit less confusing to just use same label here as well.

Reply:
I am not understanding why a normal simulate has a baseline but this is irrelevant to this PR specifically
![image](https://github.com/user-attachments/assets/82dd4450-1539-4777-adb6-2d37668c24bd)